### PR TITLE
reformat the download output

### PR DESCRIPTION
**Pull request by reidliu41**
reformat the `download` output
e.g.
```
$ ilab model download
Downloading model from Hugging Face:
    Model: instructlab/merlinite-7b-lab-GGUF@main
    Destination: /Users/user/.cache/instructlab/models

Downloading model failed with the following Hugging Face Hub error:
    (ProtocolError('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer')), '(Request ID: eb0fd877-eae3-4799-b6a2-47422213a96f)')

Downloading model failed with the following error: 1

---------------
$ ilab model download --repository=TheBloke/Mixtral-8x7B-Instruct-v0.1-GGUF --filename=mixtral-8x7b-instruct-v0.1.Q4_K_M.gguf
Downloading model from Hugging Face:
    Model: TheBloke/Mixtral-8x7B-Instruct-v0.1-GGUF@main
    Destination: /Users/user/.cache/instructlab/models

TheBloke/Mixtral-8x7B-Instruct-v0.1-GGUF requires a HF Token to be set.
Please use '--hf-token' or 'export HF_TOKEN' to download all necessary models

---------------
$ ilab model download --repository=TheBlokeMixtral-8x7B-Instruct-v0.1-GGUF --filename=mixtral-8x7b-instruct-v0.1.Q4_K_M.gguf
repository TheBlokeMixtral-8x7B-Instruct-v0.1-GGUF matches neither Hugging Face nor OCI registry format.
Please supply a valid repository.

---------------
$ ilab model download --repository docker://xx.xx.io/xx/mixtral-8x7b-instruct-v0-1 --release 1.1
Downloading model from OCI registry:
    Model: docker://xx.xx.xx/xx/mixtral-8x7b-instruct-v0-1@1.1
    Destination: /Users/user/.cache/instructlab/models

skopeo is not installed.
Please install recommended version: 1.9.0

Downloading model failed with the following error: 1

---------------
$ ilab model download --repository docker://xx.xx.xx/xx/mixtral-8x7b-instruct-v0-1 --release 1.1
Downloading model from OCI registry:
    Model: docker://xx.xx.xx/xx/mixtral-8x7b-instruct-v0-1@1.1
    Destination: /Users/user/.cache/instructlab/models

Downloading model failed with the following error:
    skopeo version 1.8.1 is lower than 1.9.0. 
    Please upgrade it.
```

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the
  [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary).
- [ ] [Changelog](https://github.com/instructlab/instructlab/blob/main/CHANGELOG.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Functional tests have been added, if necessary.
- [ ] E2E Workflow tests have been added, if necessary.

### CODE REVIEW

**Review by cdoern — APPROVED** [review]
looks nice, thanks for this

File: src/instructlab/model/download.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def download(self):
         Download specified model from Hugging Face
         """
         click.echo(
-            f"Downloading model from Hugging Face: {self.repository}@{self.release} to {self.download_dest}..."
+            f"Downloading model from Hugging Face:\n    Model: {self.repository}@{self.release}\n    Destination: {self.download_dest}"
```
Comment (danmcp) [inline src/instructlab/model/download.py:69]: Suggestion: Create a constant for the indention spaces:

DEFAULT_INDENT = "    "

**Review by danmcp — APPROVED** [review]

**Comment by reidliu41** [comment]
thanks

**Comment by reidliu41** [comment]
hi team, anyone more help to take a look? thanks.

**Comment by cdoern** [comment]
@jaideepr97  PTAL

**Review by jaideepr97 — APPROVED** [review]
LGTM thanks @reidliu41

**Pull request merged**